backend/src/helpers/jwt/extract-jwt-token/extract-jwt-token.helper.ts

```diff
@@ -0,0 +1,6 @@
+const extractJwtToken = (header: string): string => {
```

Comment:
```suggestion
const extractAuthToken = (header: string): string => {
```
don't forget to change the name of the file